extract.py

```diff
@@ -78,4 +78,6 @@ def load_into_dataframe() -> pd.DataFrame:
 
 
 if __name__ == "__main__":
-    load_into_dataframe()
+    pd.set_option('display.max_rows', None)
+    pd.set_option('display.max_columns', None)
+    print(load_into_dataframe())
```

Comment:
no need to push the print statement

Reply:
i keep accidentally pushing my local changes to extract - edited now!